doc/reference/reference_lua/net_box.rst

```diff
@@ -635,6 +622,64 @@ Below is a list of all ``net.box`` functions.
         The option can be useful if you want to pass a response through without decoding or with partial decoding.
         The usage of :ref:`MsgPack object <msgpack-object-info>` can reduce pressure on the Lua garbage collector.
 
+    .. _conn-new_stream:
+
+    .. method:: stream([options])
+
+        Create a stream.
+
+        **Example:**
+
+        .. code-block:: lua
+
+           local conn = net_box.connect('localhost:3301')
+           local conn_space = conn.space.test
+           local stream = conn:new_stream()
+           local stream_space = stream.space.test
+
+.. class:: stream
+
+    .. _net_box-stream_begin:
+
+    .. method:: begin()
+
+        Begin a transaction. Instead of the direct method, you can also use the ``call``, ``eval`` or execute methods with SQL transaction.
+           
+    .. _net_box-stream_commit:
+
+    .. method:: commit()
+
+        Commit a transaction. Instead of the direct method, you can also use the ``call``, ``eval`` or execute methods with SQL transaction.
+        
+        **Examples:**
+
+        .. code-block:: lua
+
+           -- Begin transaction
+           stream:begin()
+           -- Atomically transfer `sum` from `account1` to `account2`
+           stream.space.accounts.update({account1}, {{'-', 'amount', sum}})
+           stream.space.accounts.update({account2}, {{'+', 'amount', sum}})
```

Comment:
I have two questions for @alyapunov: 
- Shouldn't it be `stream.space.accounts:update` with a semicolon?
- Is `accounts` the name of a space that was presumably created some time in the past? (If so, I suggest that @rindblack15 adds a comment about that.)

Reply:
Fixed.